diode-server/Makefile

```diff
@@ -1,4 +1,6 @@
-SERVICES         := $(shell find ./cmd/* -type d -exec basename {} \;)
+UTILITIES        = auth_manage
+BUILD_UTILITIES  = $(addprefix build-,$(UTILITIES))
+SERVICES         := $(filter-out $(UTILITIES),$(shell find ./cmd/* -type d -exec basename {} \;))
```

Comment:
this is a little funky since it generally wants to make everything in cmd into a separate service -- this utility is marked out as separate and included in the auth service image instead.